/mechaship_navigation_node:
  ros__parameters:
    ##### 그룹핑 관련 매개변수 #####
    wall_safe_range: 5.0 # 분류 방식
    buoy_safe_range: 3.0 # 분류 방식
    ##### 도킹 관련 매개변수 #####
    image_width : 400